Source/WebKit/GPUProcess/graphics/WebGPU/RemoteGPU.cpp

```diff
@@ -194,10 +196,16 @@ void RemoteGPU::createCompositorIntegration(WebGPUIdentifier identifier)
     ASSERT(m_backing);
 
     auto compositorIntegration = m_backing->createCompositorIntegration();
+    compositorIntegration->setWebProcessID(webProcessID());
     auto remoteCompositorIntegration = RemoteCompositorIntegration::create(compositorIntegration, m_objectHeap, *m_streamConnection, identifier);
     m_objectHeap->addObject(identifier, remoteCompositorIntegration);
 }
 
+unsigned RemoteGPU::webProcessID() const
+{
+    return m_gpuConnectionToWebProcess->webProcessIdentity().taskIdToken();
```

Comment:
I think we should use the identity throughout our code, and only call `taskIdToken()` at the latest possible moment.

Reply:
`ProcessIdentity` is in `WebCore/platform` so unfortunately I think this is the latest possible moment.

We can also forward declare `ProcessIdentity` and pass it throughout in PAL but then where do we use it?

Reply:
:/ ok